content/applications/websites/website/publish/translate.rst

```diff
@@ -1,39 +1,86 @@
-===========================
-How to translate my website
-===========================
+===================
+Website translation
+===================
 
-Overview
-========
+The contents of your website pages (i.e. text strings) can be translated into different languages
+directly on your website.
 
-In addition to creating great modern websites, Odoo gives you the
-possibility to translate it in different languages.
+.. note::
+   Your website is displayed in the language that matches the visitor's browser's language, unless
+   that particular language has not been installed. In this case, the website is displayed in the
+   :ref:`default language <default-language>`. The visitor can still select another language in
+   the language menu.
 
-Process
-=======
+Installing languages
+====================
 
-Once your website is created, you have the opportunity to translate it
-in as many different languages as you want.
+To translate your website, you first have to add the required language(s):
 
-You can only translate your website manually, follow the next step.
+#. Go to your website.
+#. Scroll to the bottom of the page to the **language menu**.
+#. Click the language and select :guilabel:`Add a language`.
 
-Now go to your website. On the bottom right corner of the page, click on
-**Add a language**.
+   .. image:: translate/website-add-language.png
+      :alt: Add a language to your website.
 
-.. image:: translate/translate_website01.png
-    :align: center
+#. Click the :guilabel:`Languages` field and select the required language from the drop-down
+   list. Repeat this step for each additional language.
```

Comment:
More nitpicking. I think it's because of your automatic hard wrap on PyCharm
```suggestion
#. Click the :guilabel:`Languages` field and select the required language from the drop-down list.
   Repeat this step for each additional language.
```